Added pss panic to known issues for 1.7.0 and 1.7.1 and troubleshooting also indicated that this is fixed in release 1.7.2 (#3165)
pivotal/docs-tap: release-notes.hbs.md

- Resolved issue description.

> #### <a id='1-7-2-COMPONENT-NAME-ri'></a> v1.7.2 Resolved issues: Supply Chain Security Tools - Scan and Sacn 2.0

Reviewer: ```suggestion
#### <a id='1-7-2-scst-scan-ri'></a> v1.7.2 Resolved issues: Supply Chain Security Tools - Scan an Scan 2.0
```

> - Controller panic when `container.SecurityContext` is provided but either of fields `Capabilities` or `SeccompProfile` are left emptty is addressed.

Reviewer: ```suggestion
- Scan Controller panic when `container.SecurityContext` is configured but not all fields are provided such as  `Capabilities` or `SeccompProfile` has been resolved.
```
